playwright-e2e/pages/page-base.ts

```diff
@@ -141,4 +143,77 @@ export default abstract class PageBase implements PageInterface {
     await dob.date().locator('input[data-placeholder="DD"]').fill(date.toString());
     await dob.date().locator('input[data-placeholder="YYYY"]').fill(year.toString());
   }
+
+  /**
+   * Filling out address form.
+   * @param page
+   * @param opts
+   */
+  async fillInContactAddress(opts: {
```

Comment:
Copy/pasted `enterMailingAddress()` function here and renamed for nameing convention consistency because some other helper functions starts with `fillIn`